cookies/giles.py

```diff
@@ -624,6 +640,23 @@ def process_details(data, upload_id, username):
         _create_content_resource(resource, __text__, creator, text_uri,
                                  text_uri, content_type=text_content_type)
 
+    # Content resource for each additional file, if available.
+    for additional_file in data.get('additionalFiles', []):
+        content_type = additional_file.get('content-type')
+        uri = additional_file.get('url')
```

Comment:
it's just two lines, but they keep repeating, which means it might make sense to extract them into a method.